opentelemetry-sdk/src/opentelemetry/sdk/resources/__init__.py

```diff
@@ -282,6 +272,7 @@ def detect(self) -> "Resource":
 def get_aggregated_resources(
     detectors: typing.List["ResourceDetector"],
     initial_resource: typing.Optional[Resource] = None,
+    final_resource: typing.Optional[Resource] = None,
```

Comment:
We need the attributes from `attributes` in `Resource.create()` and we can't use `initial_resource` because they need the highest priority, so I thought this was a good solution that doesn't change the API too much.

Open to suggestions for this!